README.md

```diff
@@ -65,24 +66,26 @@ All elements can be accesses as a property of the segment via their name. The st
 segment.AREAMQZM2.k1 = torch.tensor(4.2)
 ```
 
-In order to track a beam through the segment, simply call the segment like so
+You can choose to track either a beam defined by its parameters (fast) or by its particles (precise). _Cheetah_ defines two different beam classes for this purpose and beams may be created by
 
 ```python
-outgoing_beam = segment.track(incoming_beam)
+parameter_beam = cheetah.ParameterBeam.from_twiss(beta_x=torch.tensor(3.14))
```

Comment:
That reminds me that the `ParameterBeam` does not document in its doc string what kind of distribution it draws from. Is that a Waterbag? Cannot quickly spot this in `parameter_beam.py`.

We could also mention the distribution here in the prose.